plugins/volumeFilter_test.go

```diff
@@ -287,36 +287,37 @@ func TestVolumeFilterFn(t *testing.T) {
 	accountIDs := []string{}
 
 	for _, k := range testCases {
-		t.Run(k.name, func(t *testing.T) {
-			// exactly one of the two cap values must be set
-			if k.sellBaseCapInBase == nil && k.sellBaseCapInQuote == nil {
-				assert.Fail(t, "either one of the two cap values must be set")
-				return
-			}
-
-			if k.sellBaseCapInBase != nil && k.sellBaseCapInQuote != nil {
-				assert.Fail(t, "both of the cap values cannot be set")
-				return
-			}
+		for _, action := range []queries.DailyVolumeAction{queries.DailyVolumeActionSell} {
+			t.Run(k.name, func(t *testing.T) {
+				// exactly one of the two cap values must be set
+				if k.sellBaseCapInBase == nil && k.sellBaseCapInQuote == nil {
```

Comment:
these should be called `baseCapInBase` and `baseCapInQuote` instead of `sellBaseCapInBase` and `sellBaseCapInQuote`. Better to do this in a separate PR since the overhead of this PR is already getting complex. It's ok to do this when you add the `buy` action to this for loop.